### PR TITLE
Modified get involved page to the current version of EvalAI

### PR DESCRIPTION
**Pull request by Sanji515**
**Minor changes related to css**

- Changed the link color from red to yellow and some font related css to match with the theme of current version

- Link to live demo: http://pr-136-evalai.surge.sh 

<br>

**<p align="center">Old UI Screenshot 1:**</p>

Screenshot from 2019-06-24 16-32-42

**<p align="center">Modified UI Screenshot 1:</p>**

Screenshot from 2019-07-03 19-06-02

**<p align="center">Modified UI Screenshot 2:</p>**

Screenshot from 2019-06-25 21-30-59

### CODE REVIEW

**Review by Shekharrajak — COMMENTED** [review]
I don't see much difference. Can you let us know what are the changes you did in current version of EvalAI? Make sure you write your changes in PR description properly.

**Comment by lunayach** [comment]
Hi, @Sanji515 could you respond to @Shekharrajak and add some tests before we merge this?

**Comment by Sanji515** [comment]
> Make sure you write your changes in PR description properly

@Shekharrajak @lunayach Updated the PR description, please have a look

**Comment by Sanji515** [comment]
> add some tests before we merge this

done, @lunayach please review

**Comment by Sanji515** [comment]
@lunayach @Shekharrajak please review

**Comment by lunayach** [comment]
Very minor things, but better if we don't leave them for the last moment. 

 Screenshot from 2019-06-25 14-34-50

UL list having "Get Involved", "CloudCV Website" etc. seems misaligned. (IMO)

 Screenshot from 2019-06-25 14-34-30

Should we have a border separating the paragraph above and "Report issues"? Also little too much whitespaces around the centred text. (IMO)

File: src/app/components/get-involved/get-involved.component.spec.ts [hunk 1]
```diff
@@ -35,11 +37,19 @@ describe('GetInvolvedComponent', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(GetInvolvedComponent);
+    globalService = TestBed.get(GlobalService);
+    globalServiceSpy = spyOn(globalService, 'scrollToTop');
     component = fixture.componentInstance;
+
+    expect(globalServiceSpy).not.toHaveBeenCalled();
```
Comment (lunayach) [inline src/app/components/get-involved/get-involved.component.spec.ts:44]: I'm not sure if this is helpful.

**Comment by lunayach** [comment]
`Merging #136 into master will decrease coverage by 0.01%.`

@Sanji515 if merging the code does not increase the code-coverage, we should make sure that the coverage is not decreased at least.

**Comment by Sanji515** [comment]
@lunayach I think that's the only test case can be written for this static page, please let me know me if I'm missing something

File: src/app/components/get-involved/get-involved.component.spec.ts (hunk 1 above)
Comment (Sanji515) [reply src/app/components/get-involved/get-involved.component.spec.ts:44]: hey @lunayach it's nothing wrong to have that but also it doesn't make any sense to have that, so will remove that

**Comment by lunayach** [comment]
Okay, @Sanji515. It's fine for this PR, but please take a note of it for future PR's.

**Comment by Sanji515** [comment]
@lunayach yeah I do check report but even after adding that one test the report hasn't updated, it's still the same.

**Comment by Sanji515** [comment]
> UL list having "Get Involved", "CloudCV Website" etc. seems misaligned

@lunayach hey I think footer is going to be change a/c to our current version, so there's no need to do that

> Should we have a border separating the paragraph above and "Report issues"? Also little too much whitespaces around the centred text

done and updated the PR description, please review

**Comment by Sanji515** [comment]
@lunayach please review

**Comment by Sanji515** [comment]
@lunayach can you please take a look at this?

**Comment by lunayach** [comment]
HI @Sanji515, by the border in the above comment, I meant horizontal line separating the section beginning with "Report issues" and the introductory text above it. I guess, you may have misunderstood and underlined all the headings like "Report issues", "Press" instead. Those may not be required and could be removed. (IMO)

**Comment by Sanji515** [comment]
@lunayach sorry that I misunderstood the things, I've removed the border bottom of headings and added a border above the first heading and below the intro text

Please review

File: src/app/components/get-involved/get-involved.component.scss [hunk 2]
```diff
@@ -69,10 +77,15 @@
...
+  .get-involved-container {
+    .get-involved-title {
+      margin: 0px;
+    }
+  }
   .get-involved-section {
     div {
       .get-involved-section-title {
-        margin-top: 50px;
+        margin: 0px !important;
```
Comment (lunayach) [inline src/app/components/get-involved/get-involved.component.scss:88]: Was `!important` needed? (and in other places)

Comment (Sanji515) [reply src/app/components/get-involved/get-involved.component.scss:88]: @lunayach yeah actually it is not needed here, removed from here. I've used `!important` in one other place also but there it is needed so I've not removed from there please have a look

Comment (lunayach) [reply src/app/components/get-involved/get-involved.component.scss:88]: Okay.

**Review by lunayach — APPROVED** [review]
Looks good to me. LGTM.

**Pull request merged**